aquadoggo/src/bus.rs

```diff
@@ -12,4 +15,19 @@ pub type ServiceSender = Sender<ServiceMessage>;
 pub enum ServiceMessage {
     /// A new operation arrived at the node.
     NewOperation(OperationId),
+
+    /// Message from scheduler to initiate replication on available peers.
+    InitiateReplication,
```

Comment:
Why is this a bus message? Is it needed across services?

Reply:
> Why is this a bus message? Is it needed across services?

I didn't have that in mind, I just needed to get messages from the network service -> replication service and thought this was the way to do it.... Is there a better way?

Reply:
> network service -> replication service

I only see this being used in Replication Service

Reply:
Oh yeh, sorry, I was thinking the comment included the other new variants too! Yeh, this one is only used in the Replication Service :+1:

Reply:
> I was thinking the comment included the other new variants too

Hehe yeah, the other "new variants" I actually added :-D

Reply:
> Yeh, this one is only used in the Replication Service

Cool, I think using the service bus for this is too global, it can be either a channel again or even better an async stream which wakes up every x seconds (as I've written in another comment).